test/e2e/specs/site-editor/command-center.spec.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Site editor command center', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		// Navigate to the site editor.
+		await admin.visitSiteEditor();
+	} );
+
+	test.skip( 'Open the command center and navigate to the page create page', async ( {
```

Comment:
Why do we want to add these skipped tests? Perhaps adding comments above would help?

Reply:
Because they're hidden behind a feature flag right now and we'll be removing the feature flag soon. I preferred to have the tests in place to allow us to iterate and I verified that the tests pass.

I tried enabling the feature flag using the global variable but it doesn't work because you need to trigger a "rerender". I think ideally we need utils in the e2e tests to enable and disable feature flags (navigate to the options screen and enable the flags), I'm pretty sure we had these at some point.

Reply:
Still, a comment above would be helpful for the readers (or reviewers) :).

> navigate to the options screen and enable the flags

It's [recommended](https://github.com/WordPress/gutenberg/tree/trunk/docs/contributors/code/e2e#restify-actions-to-clear-or-set-states) to create this util as a `requestUtil` to speed up the process. You are welcome to create one!